Tests/Common/Orders/Fees/AlphaStreamsFeeModelTests.cs

```diff
@@ -0,0 +1,198 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using NUnit.Framework;
+using QuantConnect.Data;
+using QuantConnect.Data.Market;
+using QuantConnect.Orders;
+using QuantConnect.Orders.Fees;
+using QuantConnect.Securities;
+using QuantConnect.Securities.Cfd;
+using QuantConnect.Securities.Forex;
+using QuantConnect.Securities.Future;
+using QuantConnect.Securities.Option;
+using QuantConnect.Tests.Common.Securities;
+
+namespace QuantConnect.Tests.Common.Orders.Fees
+{
+    [TestFixture]
+    class AlphaStreamsFeeModelTests
+    {
+        private decimal _liborRate = 0.024m;
+        private readonly IFeeModel _feeModel = new AlphaStreamsFeeModel(0.024m);
+
+        [Test]
+        public void USAEquityFeeInUSD()
+        {
+            var security = SecurityTests.GetSecurity();
+            security.SetMarketPrice(new Tick(DateTime.UtcNow, security.Symbol, 100, 100));
+
+            var fee = _feeModel.GetOrderFee(
+                new OrderFeeParameters(
+                    security,
+                    new MarketOrder(security.Symbol, 1000, DateTime.UtcNow)
+                )
+            );
+
+            Assert.AreEqual(Currencies.USD, fee.Value.Currency);
+            Assert.AreEqual((0.004m + _liborRate) * security.Price * 1000, fee.Value.Amount);
+        }
+
+        [Test]
+        public void USAFutureFee()
+        {
+            var tz = TimeZones.NewYork;
+            var security = new Future(Symbols.Fut_SPY_Feb19_2016,
+                SecurityExchangeHours.AlwaysOpen(tz),
+                new Cash("USD", 0, 0),
+                SymbolProperties.GetDefault("USD"),
+                ErrorCurrencyConverter.Instance
+            );
+            security.SetMarketPrice(new Tick(DateTime.UtcNow, security.Symbol, 100, 100));
+
+            var fee = _feeModel.GetOrderFee(
+                new OrderFeeParameters(
+                    security,
+                    new MarketOrder(security.Symbol, 1000, DateTime.UtcNow)
+                )
+            );
+
+            Assert.AreEqual(Currencies.USD, fee.Value.Currency);
+            Assert.AreEqual(1000 * 0.50m, fee.Value.Amount);
+        }
+
+        [Test]
+        public void USAOptionFee()
```

Comment:
style - feel free to ignore, but please consider more descriptive names - I tend to prefer test method names that are descriptive as to what case is being tested and what the expected result is. Reading this test name, `USAOptionFee`, all I know is that it's test option fees of the USA market, but I have no idea what the expected result is or the logic behind the result. I think something like, `GetOrderFee_Is50BasisPointsInUSD_WhenSecuityTypeIsOption` -- the usage of underscores is a recent personal style change aimed at making the test names more readable. I've lately been generally following a pattern of `<method>_<result>_when<case>` as it makes reading any test logs extremely explicit as to what was tested (`<classname>.<method>``), under what conditions `when<case>`, and the expected result `<result>`.